packages/docs/pages/lib/js.mdx

```diff
@@ -442,6 +442,23 @@ We do this using deterministic hashing to assign users a value between 0 and 1 f
 
 **Note** - If a user is excluded from an experiment due to the namespace range, the rule will be skipped and the next matching rule will be used instead.
 
+### Encrypting Features list
+
+When creating an API key, you have the option to enable encryption. When enabled, GrowthBook will encrypt the list of features in the API response.
+
+With encryption enabled, the response would be:
+
+```js
+  {
+    status: 200,
+    features: {},
+    encryptedFeatures: {
+      content: "encryptedFeatures",
+      iv: "hash"
+    }
+  }
+```
+
```

Comment:
I think you pulled the wrong changes into this PR.  Remove all the encryption stuff from this file.

In this file, we probably do want to change the `Quick Start` at the top to set `enableDevMode` to true (with a comment).  We also might want a dedicated `Dev Mode` section in this page with a link to the Chrome extension.

We also want to make similar changes the following doc files:
- packages/docs/pages/lib/react.mdx
- packages/sdk-js/README.md
- packages/sdk-react/README.md

And add `enableDevMode: true` to the code snippets within the app:
- packages/front-end/components/features/CodeSnippetModel.tsx

The basic idea is that dev mode is super useful and provides a better developer experience when getting started.  So we want to keep it on by default by including it in all of the sample code we show people.  We also want to make it super obvious that it's on, which is why we're making it an opt-in setting vs opt-out.  That way, fewer people will make the mistake of deploying "dev mode" to production.

Reply:
In addition, we probably want to add an alert box to the chrome extension.  If the devtools can't fine GrowthBook on a page, it's likely because devMode is not enabled.  The devtools is in a different repo - https://github.com/growthbook/devtools/blob/main/src/index.tsx#L31